.gitignore

```diff
@@ -1,2 +1 @@
 target
-Cargo.lock
```

Comment:
Why would you want to check in the `Cargo.lock` file as a library?

Reply:
This is needed by nix for reproducible builds.